### DISCUSSION TITLE
Retrieve list of followers/following?

### DISCUSSION THREAD

**Discussion by nawalvalliani** [davidteather/TikTok-Api#966] [Q&A]
Is there a functionality to retrieve a list of followers or following for a given account? For example, I would like to retrieve a list of who follows User A and who User A follows. I do not see discussion in the documentation regarding this nor possible endpoints to aid in retrieving this information.

Thanks.

**Comment by davidteather** [comment]
I believe you have to be logged in to view users, which this doesn't support.

**Reply by ilan1009** [reply]
Damn, got my hopes up for developing a script to see whos someones friends are (following and being followed), is there any alternative way?

**Reply by MarkZukian** [reply]
> Damn, got my hopes up for developing a script to see whos someones friends are (following and being followed), is there any alternative way?

It's impossible , unless you hack into Tiktok's Server which seem illage.

**Reply by night-0909** [reply]
> > Damn, got my hopes up for developing a script to see whos someones friends are (following and being followed), is there any alternative way?
> 
> It's impossible , unless you hack into Tiktok's Server which seem illage.

You can get follower and following list as they appear on Tiktok website, so 10000 max at the moment.
See : https://github.com/davidteather/TikTok-Api/issues/1159#issuecomment-5226851137
